### DISCUSSION TITLE
认证信息失效，云端链路将不可用，请进入 Xiaomi Home 集成页面，点击“选项”重新认证

### DISCUSSION THREAD

**Discussion by 2956066717** [XiaoMi/ha_xiaomi_home#513] [Ideas]
云端的token时间太短了，官方可否将时间配置的长点，几乎一两天就得认证一次，认证的比较频繁

Xiaomi Home 集成错误
检测到 XXXX 出现错误，请进入选项页面重新配置。

错误信息:
认证信息失效，云端链路将不可用，请进入 Xiaomi Home 集成页面，点击“选项”重新认证

**Comment by refined-fish** [comment]
已解决

**Comment by H0ne1d** [comment]
经常出现这个问题。不知道怎么解决。
